src/lib/userStorage/FeedStorage.js

```diff
@@ -485,10 +485,11 @@ export class FeedStorage {
   }
 
   async getCounterParty(feedEvent) {
-    let addressField = getEventDirection(feedEvent)
+    const addressField = getEventDirection(feedEvent)
 
     log.debug('getCounterParty:', feedEvent.data.receiptEvent, feedEvent.id, feedEvent.txType)
-    const address = get(feedEvent, `data.receiptEvent.${addressField}`)
+
+    const address = get(feedEvent, `data.receiptEvent.${addressField.toLowerCase()}`)
```

Comment:
that was the bug?

Reply:
Yep sort of, we're using `from` and `to` to do other stuff, so making it capitalize broke things.